jujugui/static/gui/src/app/app.js

```diff
@@ -928,8 +928,13 @@ YUI.add('juju-gui', function(Y) {
         <window.juju.components.DeploymentFlow
           acl={this.acl}
           addTemplate={jem.addTemplate.bind(jem)}
-          changes={changesUtils.getGroupedChanges(currentChangeSet)}
+          changesFilterByParent={
```

Comment:
It looks like you're calling `changesUtils` methods and passing the `changesUtils` object into them. It would probably be cleaner if you just passed in the `changesUtils` object into the component and then used it from there.